shared/wallets/wallet/header/index.js

```diff
@@ -57,6 +58,7 @@ const Header = (props: Props) => (
         onSendToAnotherAccount={props.onSendToAnotherAccount}
         disabled={!props.walletName}
       />
+      <Kb.Button type="Wallet" onClick={props.onRequest} label="Request" disabled={!props.walletName} />
```

Comment:
I don't think we need this, the updated receive screen with request button is in PR so people won't be without a  button for long

Reply:
Oh cool good to know; will remove.